File: CS/ConditionalFormatting/ViewModel.cs
using DevExpress.Mvvm;
using System.Collections.ObjectModel;

namespace ConditionalFormatting {
    public class ViewModel : ViewModelBase {
        public ViewModel() {
            Items = SaleDataModel.GetSales();
        }
        public ObservableCollection<SaleData> Items { get; set; }
    }
}

File: VB/ConditionalFormatting/ViewModel.vb
Imports DevExpress.Mvvm
Imports System.Collections.ObjectModel

Namespace ConditionalFormatting

    Public Class ViewModel
        Inherits ViewModelBase

        Public Sub New()
            Items = GetSales()
        End Sub

        Public Property Items As ObservableCollection(Of SaleData)
    End Class
End Namespace
